### PR TITLE
Adding MPI as optional dependency in installation.rst Fixes #2506

### LINKED ISSUE #2506: Documenting MPI Library  as optional dependency
The current installation doesn't list MPI as optional dependency. 
https://radicalpilot.readthedocs.io/en/stable/installation.html#software

We need to document this as a dependency to easier "Up and Running Process"

### PR DESCRIPTION
**Pull request by kartikmodi**

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -111,6 +111,14 @@ If you don't have access to a suitable machine with a public IP, you can open
 a ticket with RADICAL lab and we will provide a testing MongoDB instance for
 you.
 
+MPI Implementation Library (Optional)
+-------
+
+The MPI is used in launch method. There are several MPI flavours and implementations available
```
Comment (andre-merzky) [inline docs/source/installation.rst:117]: For
```
The MPI is used in launch method
```
I would suggest
```
MPI is used as a task launch method
```

**Review by andre-merzky — APPROVED** [review]
Left one comment, otherwise this looks good to me - thanks!

**Comment by mtitov** [comment]
@andre-merzky have a general comment - should we use a dedicated branch (`docs/nov21`) for any changes in documentation? (even if some section are missing there, just fill up with such new descriptions)

**Comment by AymenFJA** [comment]
> @andre-merzky have a general comment - should we use a dedicated branch (`docs/nov21`) for any changes in documentation? (even if some section are missing there, just fill up with such new descriptions)

agreed!

**Comment by andre-merzky** [comment]
@AymenFJA @mtitov : yes, we can do that.  Small PRs still make sense, but I guess they should be directed toward that documentation branch then...

**Comment by kartikmodi** [comment]
Have to redo the work, because of documentation being in a separate branch.

**Pull request closed without merging**